core/templates/pages/exploration-player-page/services/image-preloader.service.ts

```diff
@@ -253,10 +253,31 @@ angular.module('oppia').factory('ImagePreloaderService', [
       }
     };
 
+    /**
+    * Gets the dimensions of the math SVGs from the SVG filename provided.
+    * @param {string} filename - The string from which the dimensions of the
+    *                           math SVGs should be extracted.
+    */
+    var getDimensionsOfMathSvg = function(filename) {
+      var filenameComponents = filename.split('_');
+      var indexOfHeightString = filenameComponents.indexOf('height') + 1;
+      var indexOfWidthString = filenameComponents.indexOf('width') + 1;
```

Comment:
should we use regexr grouping to make it simpler and readable?

Reply:
done